ui/components/configuratorComponents/MeshModel/index.js

```diff
@@ -15,7 +15,7 @@ import AppBarComponent from './styledComponents/AppBar';
 
 import DeleteIcon from '@material-ui/icons/Delete';
 import SaveIcon from '@material-ui/icons/Save';
-import { AvatarGroup } from '@mui/material';
+import { AvatarGroup } from '@material-ui/core';
```

Comment:
Is it ideal to use core elements from mui. Since we need to move to mui V5 then again we have to revert these changes